src/main/java/dk/sdu/se_f22/searchmodule/infrastructure/SearchModuleImpl.java

```diff
@@ -52,7 +53,7 @@ public SearchHits search(String query) {
 
         SearchHits searchHits = new SearchHits();
         searchHits.setContents(List.of());
-        searchHits.setProducts(queryIndexOfType(Product.class, tokens));
+        searchHits.setProducts(queryIndexOfType(BaseProduct.class, tokens));
```

Comment:
This should be `Product.class` right?

Reply:
It shouldn't cause compile issues, since they use reflection.
So we (2.2) will adapt to the new product class on the search query once this PR has been sorted.

Reply:
I just checked it doesnt and therefore will change it to Product